### ISSUE TITLE
Media aborted due to corruption problem. 

### ISSUE THREAD

**Issue by bsloopcore** [Section508Coordinators/DHSA-TT-200-A#465]
URGENT!

## Issue Type (put a capital X between the brackets to indicate issue type)
- [ ] System Bug (the course doesn't function properly, specifically on the 12A video content)

## Description
[On the Mandatory training for section 508 Trusted tester, there is a video demo, that is a part of the training and should be completed in order to proceed with the certification process, hence this bug block the flow of the course. ]

## Location
[https://training.section508testing.net/mod/page/view.php?id=2947]

## Actual Behavior / Observation
[Media aborted due to corruption problem.  I tried it on browsers i have and the same issue occurs at few seconds after]
![image](https://user-images.githubusercontent.com/89048351/131514841-c646baca-85cb-4202-b1b7-5dd07fdde8bd.png)

## Expected Behavior / Recommendation
[Since this is mandatory for the Trusted tester Exam completion, it should play the video demo and mark the test completed during the session. ]

## Steps to Reproduce the Issue (if applicable)
[Go to the URL and play the video 12A the wait few seconds]
![Capture](https://user-images.githubusercontent.com/89048351/131515141-1d8aef58-8b27-46ac-ba21-2cc3d6d42426.PNG)
